tools/rosbag/src/player.cpp

```diff
@@ -364,6 +382,33 @@ void Player::doPublish(MessageInstance const& m) {
                     return;
                 }
                 break;
+            case ']':
+                options_.time_scale *= 1.1;
+                ROS_INFO_STREAM("time scale " << options_.time_scale);
+                time_publisher_.setTimeScale(options_.time_scale);
+                time_translator_.setTimeScale(options_.time_scale);
+                break;
+            case '[':
+                options_.time_scale *= 0.9;
+                ROS_INFO_STREAM("time scale " << options_.time_scale);
+                time_publisher_.setTimeScale(options_.time_scale);
+                time_translator_.setTimeScale(options_.time_scale);
+                break;
+            case 127: // backspace (or delete?)  
+                // TBD or set time scale to rate set at command line?
+                options_.time_scale = 1.0;
+                ROS_INFO_STREAM("time scale " << options_.time_scale);
+                time_publisher_.setTimeScale(options_.time_scale);
+                time_translator_.setTimeScale(options_.time_scale);
+                break;
+            case ',':
+                skip_time_ = true;
+                new_time_ = time_publisher_.getTime() - ros::Duration(5.0);
+                break;
+            case '.':
+                skip_time_ = true;
+                new_time_ = time_publisher_.getTime() + ros::Duration(5.0);
+                break;
```

Comment:
I meant those changes for a future pull request, but as it is I'll address the other comments and add the skip forward/back to the hotkey description.